docs/concepts/configuration/assign-pod-node.md

```diff
@@ -77,6 +77,10 @@ with a standard set of labels. As of Kubernetes v1.4 these labels are
 * `beta.kubernetes.io/os`
 * `beta.kubernetes.io/arch`
 
+**Note:** The value of these labels are cloud provider specific and are not guaranteed to be reliable.
```

Comment:
Please follow the style guide for callouts:
https://kubernetes.io/docs/home/contribute/style-guide/#note

Reply:
Replace with:
```
**Note:** Labels are specific to cloud providers and are not guaranteed to be reliable.
```

Reply:
I fixed the style, but your suggested text may confuse users thinking the existence of the labels are not guaranteed. The labels with these keys are added, but their values are not guaranteed to be consistent in all environments.

Reply:
OK--in that case, only one small change:
> The value of these labels is cloud provider specific

Reply:
Done. Thanks!